omnibus/config/software/msodbcsql18.rb

```diff
@@ -0,0 +1,50 @@
+name "msodbcsql18"
+default_version "18.3.2.1-1"
+
+dependency "unixodbc"
+
+skip_transitive_dependency_licensing true
+ship_source_offer true
```

Comment:
Does this apply here since the package is closed source?

Reply:
good catch. will get rid of this line